packages/sign-client/package.json

```diff
@@ -26,7 +26,7 @@
     "build": "run-s build:pre build:cjs build:esm build:umd",
     "test:pre": "rm -rf ./test/test.db",
     "test:run": "env TS_NODE_PROJECT=\"tsconfig.cjs.json\" mocha --timeout 10000 --exit --r ts-node/register ./test/*.spec.ts",
-    "test:concurrency": "env TS_NODE_PROJECT=\"tsconfig.cjs.json\" mocha --timeout 100000 --exit --r ts-node/register ./test/concurrency/*.spec.ts",
+    "test:concurrency": "env TS_NODE_PROJECT=\"tsconfig.cjs.json\" mocha --timeout 1900000 --exit --r ts-node/register ./test/concurrency/*.spec.ts",
```

Comment:
nitpick: what is `190000`?

Reply:
I've increased the timeout to a higher value to account for increased latency to allow longer tests.
This particular value seemed to work reliably when running large n sockets/messages with local and dev/prod relays